### PR TITLE
Dx 934 payment instrument changes

### PR DESCRIPTION
**Pull request by marinafrydenb**

### CODE REVIEW

File: _includes/abort-reference.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-{% assign instrument = include.instrument | default: "selecting the payment instrument" %}
+{% assign api_resource = include.api_resource | default: "selecting the payment instrument" %}
```
Comment (NullableInt) [inline _includes/abort-reference.md:1]: Maybe change the default text here?
```suggestion
{% assign api_resource = include.api_resource | default: "api-resource-not-set" %}
```

File: _includes/languages.md [hunk 2]
```diff
@@ -1,22 +1,22 @@
-{%- assign payment_instrument = include.payment_instrument -%}
-{%- assign payment_instrument_title = payment_instrument | capitalize -%}
+{%- assign documentation_section = include.documentation_section -%}
```
Comment (NullableInt) [inline _includes/languages.md:1]: Shouldn't this still be `payment_instrument?

File: _includes/payee-reference.md [hunk 3]
```diff
@@ -1,8 +1,8 @@
-{% assign payment_instrument = include.payment_instrument | default: true %}
+{% assign api_resource = include.api_resource | default: true %}
```
Comment (NullableInt) [inline _includes/payee-reference.md:1]: This can still be `payment_instrument` cant it?

File: _includes/field-description-language.md [hunk 4]
```diff
@@ -1,5 +1,5 @@
-{%- assign payment_instrument = include.payment_instrument -%}
-{% case payment_instrument %}
+{%- assign documentation_section = include.documentation_section -%}
```
Comment (NullableInt) [inline _includes/field-description-language.md:1]: Shouldn't this still be `payment_instrument`?

File: _includes/transaction-response.md [hunk 5]
```diff
@@ -1,4 +1,4 @@
-{% assign payment_instrument = include.payment_instrument | default: "test two here" %}
+{% assign api_resource  = include.api_resource  | default: "test two here" %}
```
Comment (NullableInt) [inline _includes/transaction-response.md:1]:
```suggestion
{% assign api_resource = include.api_resource | default: "test two here" %}
```
Your search-and-replace adds a ` ` more than needed.

File: _includes/transaction-response.md [hunk 6]
```diff
@@ -9,7 +9,7 @@
 {% endif %}
 
 The created `{{ transaction }}` resource contains information about the
-`{{ transaction }}` transaction made against a `{{ payment_instrument }}` payment.
+`{{ transaction }}` transaction made against a `{{ api_resource  }}` payment.
```
Comment (NullableInt) [inline _includes/transaction-response.md:12]:
```suggestion
`{{ transaction }}` transaction made against a `{{ api_resource }}` payment.
```

File: _includes/transaction-response.md [hunk 7]
```diff
@@ -19,9 +19,9 @@ HTTP/1.1 200 OK
 Content-Type: application/json
 
 {
-    "payment": "/psp/{{ payment_instrument }}/payments/{{ page.payment_id }}",
+    "payment": "/psp/{{ api_resource  }}/payments/{{ page.payment_id }}",
```
Comment (NullableInt) [inline _includes/transaction-response.md:22]:
```suggestion
    "payment": "/psp/{{ api_resource }}/payments/{{ page.payment_id }}",
```
Do a search-and-replace for `  }}` and replace it with ` }}`

File: _includes/transaction-response.md [hunk 8]
```diff
@@ -48,18 +48,18 @@ Content-Type: application/json
             "amount": 1000,
             "vatAmount": 250,
             "description": "Test transaction",
-            "payeeReference": "AH123456", {% if payment_instrument == "invoice" %}
+            "payeeReference": "AH123456", {% if api_resource  == "invoice" %}
             "receiptReference": "AH12355", {% endif %}
             "failedReason": "",
             "isOperational": false,
-            "operations": [{% if payment_instrument == "swish" and mcom == true %}
+            "operations": [{% if api_resource  == "swish" and mcom == true %}
```
Comment (NullableInt) [inline _includes/transaction-response.md:55]:
```suggestion
            "operations": [{% if api_resource == "swish" and mcom == true %}
```
Same here, do a search-and-replace for `  ==` and replace with ` ==`

File: _includes/field-description-description.md [hunk 9]
```diff
@@ -1,12 +1,12 @@
-{%- assign payment_instrument = include.payment_instrument -%}
-{%- if payment_instrument == "mobilepay"-%}
-    {%- assign payment_instrument ="mobile-pay" -%}
+{%- assign documentation_section = include.documentation_section -%}
+{%- if documentation_section == "mobilepay"-%}
+    {%- assign documentation_section ="mobile-pay" -%}
```
Comment (NullableInt) [inline _includes/field-description-description.md:3]: The purpose of this task is to remove such checks, if needed you should pass to parameters to the include, one for the `documentation_section` and one for the `api_resource`

File: payments/card/other-features.md [hunk 10]
```diff
@@ -26,7 +26,8 @@ sidebar:
 Credit Card. This section has extented code examples and features that were not
 covered by the other subsections." %}
 
-{% include payment-resource.md payment_instrument="card" show_status_operations=true %}
+{% include payment-resource.md api_resource="card" documentation_section="card"
```
Comment (asbjornu) [inline payments/card/other-features.md:29]:
```suggestion
{% include payment-resource.md api_resource="card" documentation_section="creditcard"
```

File: _includes/transaction-list-response.md [hunk 11]
```diff
@@ -1,5 +1,6 @@
-{% assign payment_instrument = include.payment_instrument | default: "paymentorder" %}
+{% assign api_resource = include.api_resource  | default: "paymentorder" %}
```
Comment (asbjornu) [inline _includes/transaction-list-response.md:1]:
```suggestion
{% assign api_resource = include.api_resource  | default: "paymentorders" %}
```

File: README.md [hunk 12]
```diff
@@ -130,14 +130,14 @@ Some includes can also take variables, if they do this is defined at the
 beginning of the file using the following syntax.
 
 ```markdown
-{% assign instrument = include.payment-instrument | default: "paymentorder" %}
+{% assign api_resource = include.api_resource | default: "paymentorder" %}
```
Comment (asbjornu) [inline README.md:133]:
```suggestion
{% assign api_resource = include.api_resource | default: "paymentorders" %}
```

**Pull request merged**